.github/workflows/ci.yml

```diff
@@ -33,7 +36,12 @@ jobs:
     outputs:
       latest: ${{ steps.set-matrix.outputs.requireds }}
     steps:
-      - uses: ljharb/actions/node/matrix@main
+      - name: Harden the runner (Audit all outbound calls)
+        uses: step-security/harden-runner@ec9f2d5744a09debf3a187a3f4f675c53b671911 # v2.13.0
+        with:
+          egress-policy: audit
+
+      - uses: ljharb/actions/node/matrix@7f214d8efdbdcefc96ad9689663ef387a195deec # main
         id: set-matrix
         with:
           versionsAsRoot: true
```

Comment:
We should explicitly matrix, not use a dep

Reply:
I think this is okay: Jordan is a collaborator (and a member of the migrations team), a TC39 delegate (or something higher?), and very well established in this space (he's the author of nvm). If he really wanted to eff us over, he could 😉

Reply:
It's actually not though because it's dynamic, which means no maintenance for us